pkg/bpf/map_linux.go

```diff
@@ -933,7 +994,7 @@ func (m *Map) DeleteAll() error {
 
 	nextKey := make([]byte, m.KeySize)
 
-	if m.cache != nil {
+	if m.withValueCache {
```

Comment:
I couldn't figure out, does `DeleteAll()` update the metric?

Reply:
It didn't. Added.